app/passport/views.py

```diff
@@ -0,0 +1,40 @@
+    abi = json.loads(abi)
+    contract = w3.eth.contract(contract_address, abi=abi)
+
+    #_hash = sp.getMessageHash(player, 0, tokenURI, nonce);
+    message = contract.functions.getMessageHash(player, 0, tokenURI, nonce).call()
+    private_key = settings.PASSPORT_PK
+    #signature = await accounts[0].signMessage(ethers.utils.arrayify(hash));
+    signed_message = w3.eth.account.signHash(message.hex(), private_key=private_key).signature.hex()
```

Comment:
for some reason this signed message isnt working in the contract.

Reply:
https://github.com/gitcoinco/gitcoinpassport/blob/master/test/GitcoinPassport.js#L37-L38

the ethers.js signature works though.  i wonder if im using the wrong signing message!

Reply:
here is the line that is failing https://github.com/gitcoinco/gitcoinpassport/blob/master/contracts/GitcoinPassport.sol#L19-L22